app/assets/stylesheets/campaigns.scss

```diff
@@ -0,0 +1,98 @@
+.block {
```

Comment:
Styles like `.block`, `.image`, `.title`, `.flash-message` etc looks like some general styles that could be used for other pages as well. You should write them in a separate file like general.scss, in that case others could use them in other pages.